### DISCUSSION TITLE
How to create a flow to forward MQTT data to Supabase?

### DISCUSSION THREAD

**Discussion by jpiwek** [emqx/emqx#13675] [Q&A]
Hi,

I would like to create a flow which takes MQTT data -> connects to Supabase via HTTPS REST API -> stores data.
Any suggestion on how to incorporate it with EMQX flow?

Thanks

**Comment by thalesmg** [comment]
Hi, you can use a HTTP Data Integration to send data over HTTP(S): https://docs.emqx.com/en/emqx/latest/data-integration/webhook.html

**Reply by jpiwek** [reply]
Thanks. 
That's a great idea. I tried Flow Designer with Message Source and HTTP Server sink.
By the way. I have EMQX MQTT Docker image running on my Raspi 5.
